exampleSite/content/blog/v0.10.zh-cn.md

```
@@ -0,0 +1,247 @@
+---
+
+Hextra v0.10.0 是一个重大版本更新，包含多项新功能、架构升级和使用体验优化。
+
+<!--more-->
+
+本次更新还包含了来自 10 位 [新贡献者](#contributors) 的代码提交，并解决了社区长期期待的功能需求。
+
+## 升级指南
+
+> [!IMPORTANT]
+> **破坏性变更**：此版本包含多项不兼容改动。升级前请仔细阅读检查清单和 [迁移指南](#migration-guide)。
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)